### PR TITLE
Added migration

### PR DESCRIPTION
**Pull request by ValdarRudman**

### CODE REVIEW

File: ormconfig.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "type": "postgres",
+    "host": "localhost",
+    "port": 5432,
+    "username": "postgres",
+    "password": "postgres",
+    "database": "timesheetsnew",
```
Comment (SeanCondon) [inline ormconfig.json:8]: How does this relate to `.env` file? Are these repeated?
Also are we still on `timesheetsnew`

The other thing I saw with the ansible deployment is that the username and password defaults are `timesheets_db_user` and `timesheets_db_pass`
https://gerrit.opencord.org/plugins/gitiles/ansible/role/timesheets/+/refs/heads/master/defaults/main.yml

File: package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
     "@nestjs/passport": "7.1.0",
     "@nestjs/platform-express": "7.4.4",
     "@nestjs/swagger": "4.6.1",
-    "@nestjs/typeorm": "7.1.4",
+    "@nestjs/typeorm": "^7.1.4",
```
Comment (SeanCondon) [inline package.json:31]: I think you can leave out the `^` as before

File: package.json [hunk 3]
```diff
@@ -43,12 +43,12 @@
     "passport-jwt": "4.0.0",
     "pdf-lib": "1.11.1",
     "pdf2pic": "2.1.2",
-    "pg": "8.4.0",
+    "pg": "^8.4.0",
```
Comment (SeanCondon) [inline package.json:46]: same here

File: src/migrations/1602486053913-createTables.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class createTables1602486053913 implements MigrationInterface {
+    name = 'createTables1602486053913'
```
Comment (SeanCondon) [inline src/migrations/1602486053913-createTables.ts:20]: did it generate this for you? nice

**Review by SeanCondon — APPROVED** [review]
A few small comments from me.

Not sure I understand it, but I presume there is more to it to come

File: ormconfig.json (hunk 1 above)
Comment (ValdarRudman) [reply ormconfig.json:8]: N cant access migrations from the app.module. Had to move to a json file to get it to work.
I will be removing the database name and all that soon, as we have a timesheets email, and these should be put in after the pull.

I don't know anything about the ansible db username and pass. I would think these would change when we have postgreSQL server setup, or maybe this is what Zack chose.

File: package.json (hunk 2 above)
Comment (ValdarRudman) [reply package.json:31]: I dont know why there are 2 of the same modules. I didnt add any for this.

File: package.json (hunk 3 above)
Comment (ValdarRudman) [reply package.json:46]: I am so confused as to why these were added.

File: src/migrations/1602486053913-createTables.ts (hunk 4 above)
Comment (ValdarRudman) [reply src/migrations/1602486053913-createTables.ts:20]: Ya, I have the command in the readme, will be apart of the readme branch. BTW, I will be removing the add modules in the readme branch

Comment (ValdarRudman) [reply src/migrations/1602486053913-createTables.ts:20]: NVM not in it haha

**Pull request merged**